test/languages.ts

```diff
@@ -830,3 +830,88 @@ export const DartLanguage: Language = {
   quickTestRendererOptions: [],
   sourceFiles: ["src/Language/Dart.ts"]
 };
+
+export const PikeLanguage: Language = {
+  name: "pike",
+  base: "test/fixtures/pike",
+  runCommand(sample: string) {
+    return `pike main.pike \"${sample}\"`;
+  },
+  diffViaSchema: false,
+  skipDiffViaSchema: [],
+  allowMissingNull: true,
+  features: ["union"],
+  output: "TopLevel.pmod",
+  topLevel: "TopLevel",
+  skipJSON: [
+    "github-events.json",
```

Comment:
Why are all of these skipped?

Reply:
Un-skipped most of them, described reasons for remaining ones.